### ISSUE TITLE
Use location.hash to allow external linking in Muledump

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#74]
If I want to tell a person to edit a specific setting, or go to backups manager, or whatever, it would be nice if I could send them:

```https://jakcodex.github.io/muledump/muledump.html#settings-accountLoadDelay```

or just

```muledump.html#settings-accountLoadDelay```

And it will automatically open the UI to the specified location.

**Comment by jakcodex** [comment]
The follow hashes are now supported:

#### State: any

```
#accountsjs
#accountsjs-upload
#about
#backups-upload
#backups-restore
```

#### State: `setuptools.state.loaded===true`

```
#accounts
#accounts-mass, #accounts-massSwitch
#accounts-export
#groups
#groups-create
#groups-select-all
#groups-select-enabled
#groups-select-disabled
#groups-enableAll
#groups-disableAll
#backups-create
```

**Comment by jakcodex** [comment]
Released in v9.0.0
